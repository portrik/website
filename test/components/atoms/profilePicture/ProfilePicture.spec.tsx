import { render } from '@testing-library/react';

import { ProfilePicture } from '@components';

describe('ProfilePicture', () => {
	test('Should Render ProfilePicture', () => {
		const picture = render(<ProfilePicture />);

		expect(picture.baseElement).toMatchSnapshot();
	});
});
