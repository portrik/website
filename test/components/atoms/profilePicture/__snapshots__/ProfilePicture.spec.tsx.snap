// Vitest Snapshot v1

exports[`ProfilePicture > Should Render ProfilePicture 1`] = `
<body>
  <div>
    <img
      alt="Portrait of Patrik Dvořáček"
      class="_profile-picture_641c96"
      src="/src/assets/Me.jpg"
    />
  </div>
</body>
`;
